app/javascript/pages/index.tsx

```diff
@@ -36,7 +36,7 @@ const HomePage = (_props: HomePageProps) => {
       />
       <div className="homepage-extra mt-2">
         <ActionBlock
-          header={t("welcome.newListingEmailAlert")}
+          header={<Heading priority={2}>{t("welcome.newListingEmailAlert")}</Heading>}
```

Comment:
Just curious, is this change due to changes with Bloom UIC?

Reply:
Yes